src/test/resources/application-local.properties

```diff
@@ -1,6 +1,6 @@
 test.browser=chrome
 base.url=http://localhost:8080
-test.headless=true
+test.headless=false
```

Comment:
Again, this should not be committed, this change is for local testing only.

Reply:
done the changes